### DISCUSSION TITLE
Where can I find latest documentation?

### DISCUSSION THREAD

**Discussion by dongyi-cai-windsab** [oxyplot/oxyplot#1967] [Q&A] [answered]
https://oxyplot.readthedocs.io/en/master/#

This documentation shows it's 8 year old, and I have trouble getting this one shows on the page working:

https://oxyplot.readthedocs.io/en/master/getting-started/hello-wpf-xaml.html

Because `oxy:Plot` doesn't exist.

Also I'm looking for code or xaml example for this section:

https://oxyplot.readthedocs.io/en/latest/models/series/LineSeries.html#data

LineSeries -> data, how to exactly setup ItemSource for example.

**Comment by VisualMelon** [comment, marked as answer]
Read the docs is the latest, and yes it needs updating: I'm intending to do that soon, but strapped for time at the moment.

`Plot` has been moved to oxyplot-contrib: https://github.com/oxyplot/oxyplot-contrib

The example library is full of useful examples; e.g. see https://github.com/oxyplot/oxyplot/blob/develop/Source/Examples/ExampleLibrary/Series/LineSeriesExamples.cs#L320 for an example of using an `ItemsSource` with a `PlotModel`. oxyplot-contrib has it's own examples, e.g. for `ItemsSource` with a `Plot`: https://github.com/oxyplot/oxyplot-contrib/blob/develop/Source/Examples/WPF/WpfExamples/Examples/BindingDemo/MainWindow.xaml

**Comment by dongyi-cai-windsab** [comment]
OK, then I have further question:

what's oxyplot and oxyplot-contrib? what's the difference?

**Reply by VisualMelon** [reply]
OxyPlot-contrib is for stuff that doesn't fit in the core libraries, and includes the WPF-XAML provision as it is too much effort to expect regular contributors to maintain (so instead someone fixes it up about once per release)

**Reply by dongyi-cai-windsab** [reply]
Thanks for the answer.
